lua/nvim-tree/fs.lua

```diff
@@ -57,8 +57,13 @@ function M.create(node)
     }
   end
 
+  local node_is_open = (
+    vim.g.nvim_tree_create_in_closed_folder == 0
+    and node.open or false
```

Comment:
this feature is not working and breaks default behavior.
`local node_is_open = vim.g.nvim_tree_create_in_closed_folder == 1 or node.open` should be more appropriate if i understand the feature correctly.
With you condition, setting `0` or `1` for `nvim_tree_create_in_closed_folder` does not change anything and always creates the file outside of the folder.

Reply:
Apologies, sleep deprivation has gotten the better of me again. Pushing a fix in a few!